kgraphql-ktor/src/main/kotlin/com/apurebase/kgraphql/KtorFeature.kt

```diff
@@ -61,6 +61,12 @@ class GraphQL(val schema: Schema) {
     }
 
     class FeatureInstance(featureKey: String = "KGraphQL") : Plugin<Application, Configuration, GraphQL> {
+        companion object {
+            private val playgroundHtml: ByteArray? by lazy {
+                KtorGraphQLConfiguration::class.java.classLoader.getResource("playground.html")?.readBytes()
```

Comment:
Usually there should be a warning logged but we don't have proper logging setup yet (cf. https://github.com/stuebingerb/KGraphQL/issues/117).